azure-pipelines.sds.pr-release.yml

```diff
@@ -5,7 +5,7 @@ resources:
     - repository: azTemplates
       type: github
       name: hmcts/azure-devops-templates
-      ref: master
+      ref: feature_sa_fix
```

Comment:
Temporary fix provided by DevOps for the Docker Compose issue following an update from Microsoft. Can be reverted once Microsoft have rolled the update back